### ISSUE TITLE
deno panic

### ISSUE THREAD

**Issue by Pietrek14** [denoland/deno#14850]
```
Platform: windows x86_64
Version: 1.21.0
Args: ["C:\\ProgramData\\chocolatey\\lib\\deno\\deno.exe", "run", "--allow-all", "--watch", "index.ts"]  

thread 'main' panicked at 'assertion failed: self.wr_state >= State::TlsClosed', ext\net\ops_tls.rs:426:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Unfortunately, I wasn't able to reproduce the bug. I was using Oak by the way.

**Comment by littledivy** [comment]
You are using an older version of Deno. This bug was fixed in 1.21.3 https://github.com/denoland/deno/pull/14547

**Comment by cjihrig** [comment]
Closing since this was fixed.
